src/ngraph/runtime/backend_manager.cpp

```diff
@@ -68,6 +70,10 @@ vector<string> runtime::BackendManager::get_registered_backends()
 
 shared_ptr<runtime::Backend> runtime::BackendManager::create_backend(const std::string& config)
 {
+#ifdef NGRAPH_INTERPRETER_STATIC_LIB_ENABLE
```

Comment:
@pruthviIntel , we need to register CPU backend as well if `NGRAPH_CPU_STATIC_LIB_ENABLE` is set. 
@rkimballn1 , do we still need to execute the logic below if the backend is already statically linked ?

Reply:
https://github.com/NervanaSystems/ngraph/blob/6e019518b22633093e7b0577a2f503d35375b056/src/ngraph/runtime/cpu/cpu_backend.cpp#L63 -#L73 we are using static object to do that for both STATIC and SHARED cpu_backend